src/Tizen.NUI/src/public/Key.cs

```diff
@@ -166,6 +166,20 @@ public string KeyPressed
             }
         }
 
+        /// <summary>
+        /// Get the actual string returned that should be used for input editors.
+        /// </summary>
+        /// <returns>The key string</returns>
+        /// This will be public opened in tizen_5.5 after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string KeyString
```

Comment:
Hello, private keyString is set/get property, but I can not understand why this KeyString is only get property?

Reply:
yes... I think it would be better to make them as same for set/get properties. one is only get but the other is set/get... this looks not good... 
OR if you think "private keyString" will be used even in the future, then it need be "internal keyString", I think. because we can give internal API access permission to specific module by using friend-assembly.

Reply:
Added 'Set' property in KeyString, too. :)